roles/satellite-clone/vars/satellite_6.9.yml

```diff
@@ -0,0 +1,16 @@
+---
+satellite_package: satellite
+satellite_installer_cmd: satellite-installer
+satellite_scenario: satellite
+capsule_puppet_module: foreman-proxy
+satellite_installer_options: "--foreman-ipa-authentication false --reset-puppet-server-ssl-chain-filepath --disable-system-checks"
+satellite_upgrade_options: "--disable-system-checks"
+verify_rake_task: reimport
+selinux_packages:
```

Comment:
We also have pulpcore-selinux now. Should that be listed as well?

Reply:
I think it makes sense to add it. We needed to ensure these were installed beforehand because of https://projects.theforeman.org/issues/30506

Reply:
I see we didn't add candlepin-selinux either. perhaps let's just see if it works as-is?

Reply:
I don't know satellite-clone that well. If it runs the installer, that should generally take care of installing the packages, right?

Reply:
That was also something I suspected.